derive/src/error.rs

```diff
@@ -0,0 +1,100 @@
+use std;
+
+use json::JsonError;
+
+/// The result type for this procedural macro.
+pub(crate) type Result<T> = std::result::Result<T, Error>;
```

Comment:
no reason to use `(crate)` notation here, it is not reexported on the main mod level anyway, right?

Reply:
True! :)